lambda-buffers-codegen/app/LambdaBuffers/Codegen/Cli/GenRust.hs

```diff
@@ -47,3 +53,19 @@ readRustConfig f = do
       logError "" $ "Invalid Rust configuration file " <> f
       exitFailure
     Just cfg -> return cfg
+
+readPackages :: FilePath -> IO RS.PkgMap
+readPackages f = do
+  fExists <- doesFileExist f
+  unless
+    fExists
+    ( do
+        logError "" $ "Provided Rust Codegen package manifest file doesn't exists: " <> f
```

Comment:
Remove ""?

Reply:
I think this is necessary:
https://github.com/mlabs-haskell/lambda-buffers/commit/ff570a79a0899e223e8c57070324e91631049185